staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -67,6 +69,13 @@ import (
 	"k8s.io/apiextensions-apiserver/pkg/registry/customresource/tableconvertor"
 )
 
+const (
+	// This version is being used instead of any provided version by the CRD author to make sure
+	// the list object will go through conversion. Listed object may have different versions than
+	// request or storage version. This name should be unique.
+	CrdListVersion = "crd_list_version_A173D508-268B-4654-BEF2-2A943F379F9C"
```

Comment:
😱

Reply:
this is not used, is it?

Reply:
oh, yes, this is not used. I should have removed it. Your reaction is priceless though :)

Reply:
https://imgs.xkcd.com/comics/random_number.png